src/components/routes/Host/ListingForm/ListingInfoForm/ListingInfoForm.tsx

```diff
@@ -125,46 +141,74 @@ const ListingInfoForm = (props: FormikProps<ListingInput>): JSX.Element => {
             <Svg className="suffix" src="utils/carat-down" />
           </label>
         </SelectBoxWrapper>
+        <ErrorMessageWrapper>
+          <ErrorMessage name="country" />
+        </ErrorMessageWrapper>
       </div>
 
       <div className="form-item map-preview">
         <InputLabel>Location Preview on Map</InputLabel>
         <GoogleMaps
           address={address}
           getCoordinates={({ lat, lng }) => {
-            if ((lat || lat === 0) && (values.lat || values.lat === 0) && Math.abs(lat - values.lat) > LAT_LNG_EPSILON) {
+            if (
```

Comment:
is this the same as below? can you assign it to a function? 
``isCoordinateValid(lat)``

Reply:
refactored out 👍